charts/galoy-pay/values.yaml

```diff
@@ -9,6 +9,7 @@ service:
   type: ClusterIP
 graphqlHostname: domain.example
 graphqlHostnameInternal: service-name.namespace.svc.cluster.local
+websocketUrl: wss://ws.domain.example/graphql
```

Comment:
Should we maybe call this graphqlWebsocketUrl? Seems a bit more precise and inline with the other vars